shell/packages/sandstorm-db/db.js

```diff
@@ -1809,6 +1770,58 @@ _.extend(SandstormDb.prototype, {
     });
     return readyApps.count() === packageIds.length;
   },
+
+  isReferralEnabled: function () {
+    // This function is a bit weird, in that we've transitioned from
+    // Meteor.settings.public.quotaEnabled to DB settings. For now,
+    // Meteor.settings.public.quotaEnabled implies bothisReferralEnabled and isQuotaEnabled are true.
+    return Meteor.settings.public.quotaEnabled;
+  },
+
+  isQuotaEnabled: function () {
+    const setting = this.collections.settings.findOne({ _id: "quotaEnabled" });
```

Comment:
Can you skip this database query if Meteor.settings.public.quotaEnabled is true?